DOC-278 Updated creating nodes locally in deployment guide (#493)
corda/corda-docs-portal: content/en/platform/corda/4.6/enterprise/operations/deployment/generating-a-node.md

}
```

#### Optional configuration

* `notary` &lt;config&gt; - use this configuration option to specify the node as a Notary node. **Required**> for Notary nodes. For more information, see [Notary](../setup/corda-configuration-fields.html#notary).

* `devMode` &lt;boolean&gt; - use this configuration option to enable development mode when you set its value to `true`. For more information, see [devMode](../setup/corda-configuration-fields.html#devmode). For example:

```kotlin
devMode true
```

* `rpcUsers` &lt;list&gt; - use this configuration option to set the RPC users for the node. For more information, see [rpcUsers](../setup/corda-configuration-fields.html#rpcusers). You can use arbitrary values in this configuration block - "incorrect" settings will not cause a DSL error. An example follows below:

```kotlin
rpcUsers = [[ user: "user1", "password": "test", "permissions": ["StartFlow.net.corda.flows.MyFlow"]]]
```

> * `configFile` &lt;string&gt; - use this configuration option to generate an extended node configuration. For more information, see [extended node configuration](#optional-configuration). For example:

Reviewer: ```suggestion
* `configFile` &lt;string&gt; - use this configuration option to generate an extended node configuration. For more information, see [extended node configuration](../../node/setup/corda-configuration-file.md). For example:
```

```kotlin
configFile = "samples/trader-demo/src/main/resources/node-b.conf"
```

* `sshdPort` &lt;integer&gt; - use this configuration option to specify the SSH port for the Docker container. This will be mapped to the same port on the host.  If `sshdPort` is specified, then that port must be available on the host and not in use by some other service. If `sshdPort` is not specified, then a default value will be used for the SSH port on the container. Use the `docker port <container_name>` command to check which port has been allocated on the host for your container. For more information, see [sshd](../setup/corda-configuration-fields.html#sshd). For example:

```kotlin
sshd {
  port = 2222
}
```

You can extend the `deployNodes` task with more `node {}` blocks to generate as many nodes as necessary for your application.

{{< warning >}}
When adding nodes, make sure that there are no port clashes!
{{< /warning >}}

> To extend node configuration beyond the properties defined in the `deployNodes` task, use the `configFile` property with the file path (relative or absolute) set to an additional configuration file. This file should follow the standard [Node configuration](../setup/corda-configuration-file.md) format of `node.conf`. The properties set there will be appended to the generated node configuration.

Reviewer: ```suggestion
To extend node configuration beyond the properties defined in the `deployNodes` task, use the `configFile` property with the file path (relative or absolute) set to an additional configuration file. This file should follow the standard [Node configuration](../../node/setup/corda-configuration-file.md) format of `node.conf`. The properties set there will be appended to the generated node configuration.
```

}
```

#### Package namespace ownership

> To configure [package namespace ownership](env-dev.html#package-namespace-ownership), use the optional `networkParameterOverrides` and `packageOwnership` blocks, in a similar way to how the configuration file is used by the [Network Bootstrapper](../../network-bootstrapper.md) tool. For example:

Reviewer: ```suggestion
To configure [package namespace ownership](../../node/deploy/env-dev.html#package-namespace-ownership), use the optional `networkParameterOverrides` and `packageOwnership` blocks, in a similar way to how the configuration file is used by the [Network Bootstrapper](../../network-bootstrapper.md) tool. For example:
```
